asistente_ladm_col/gui/supplies/dlg_missing_snc_supplies.py

```diff
@@ -0,0 +1,309 @@
+                            self.logger.clear_status()
+                            res_gpkg, msg_gpkg = self.package_results(self.output_etl_missing_cobol)
+                            if res_gpkg:
+                                self.generate_excel_report()
+                                if not self.custom_feedback.isCanceled():
+                                    self.progress.setValue(self.progress_maximum)
+                                    self.buttonBox.clear()
+                                    self.buttonBox.setEnabled(True)
+                                    self.buttonBox.addButton(QDialogButtonBox.Close)
+
+                                    msg = QCoreApplication.translate("Asistente-LADM-COL",
+                                        "Missing supplies report successfully generated in folder <a href='file:///{normalized_path1}'>{path1}</a>! The output Geopackage database can be found in <a href='file:///{normalized_path2}'>{path2}</a>").format(
+                                        normalized_path1=normalize_local_url(self.xlsx_path),
+                                        path1=self.xlsx_path,
+                                        normalized_path2=normalize_local_url(self.gpkg_path),
+                                        path2=self.gpkg_path)
+                                    self.logger.clear_status()
+                                    self.logger.success_msg(__name__, msg)
+                                else:
+                                    self.initialize_feedback()  # Get ready for an eventual new execution
+                                    self.progress_base = 0
+                                    self.logger.clear_status()
+
+                                self._running_tool = False
+                            else:
+                                # User could have canceled while running the second algorithm
+                                if self.custom_feedback.isCanceled():
+                                    self.initialize_feedback()  # Get ready for an eventual new execution
+                                    self.progress_base = 0
+                                    self._running_tool = False
+                                else:
+                                    self.show_message(msg_gpkg, Qgis.Warning)
+                        else:  # User canceled in the first algorithm
+                            self.initialize_feedback()
+                            self.progress_base = 0
+                            self._running_tool = False
+                            self.logger.clear_status()
+                    else:
+                        self.show_message(msg_gdb, Qgis.Warning)
+                else:
+                    self.show_message(msg_csv, Qgis.Warning)
+
+            self.set_gui_controls_enabled(True)
+
+    def validate_files_in_folder(self):
+        """
+        This function allows verifying if gpkg path or xlsx path exists.
```

Comment:
Same comment as in dlg_missing_cobol_supplies.py (validate_files_in_folder function).

Reply:
Changed